infrastructure/terraform/gcp/gke/README.md

```diff
@@ -0,0 +1,27 @@
+# GKE Cluster module
+
+Create GKE clusters
+
+## Known Issues
+
+- Currently, the cluster cannot be update after creation due to https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/issues/1876
+
+## Usage
+
+This modules assume you have the permissions required (given by user-access module) and an existing VPC (created by network module).
+
+To create multiple clusters, add a new file under the `vars/` folder and use the cluster name as the file name.
```

Comment:
not I understand why someone would want to create multiple clusters in the context of this project. I am missing in this section a step by step procedure describing how to update the infrastructure.

Reply:
If something needs to be tested in the future that could impact the current cluster, creating a new one to test and validate changes will be useful and reduce the risk of downtime on the current cluster.